### PR TITLE
Change corp industry jobs productTypeName to char

### PR DESCRIPTION
**Pull request by amclain**
This PR updates the `productTypeName` field in `/corp/IndustryJobs.xml.aspx` and `/corp/IndustryJobsHistory.xml.aspx` to type `char`.

### CODE REVIEW

File: docs/xmlapi/corporation/corp_industryjobs.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ Retrieve unfinished corporation industry jobs.
         <tr>
             <td></td>
             <td>productTypeName</td>
-            <td><strong>int</strong></td>
+            <td><strong>char</strong></td>
```
Comment (ccp-zoetrope) [inline docs/xmlapi/corporation/corp_industryjobs.md]: String is more approriate.

File: docs/xmlapi/corporation/corp_industryjobshistory.md [hunk 2]
```diff
@@ -232,7 +232,7 @@ Retrieve finished corporation industry jobs.
         <tr>
             <td></td>
             <td>productTypeName</td>
-            <td><strong>int</strong></td>
+            <td><strong>char</strong></td>
```
Comment (ccp-zoetrope) [inline docs/xmlapi/corporation/corp_industryjobshistory.md]: Again, string is more approriate.

File: docs/xmlapi/corporation/corp_industryjobs.md (hunk 1 above)
Comment (amclain) [inline docs/xmlapi/corporation/corp_industryjobs.md]: No problem. I've seem both `char` and `string` used throughout the docs; didn't know if one was preferred over the other.

Updated in b27829a.

File: docs/xmlapi/corporation/corp_industryjobshistory.md (hunk 2 above)
Comment (amclain) [inline docs/xmlapi/corporation/corp_industryjobshistory.md]: Updated in b27829a.

**Pull request merged**